yt/yt/server/job_proxy/environment.cpp

```diff
@@ -306,10 +306,10 @@ class TPortoUserJobEnvironment
 
         std::vector<TDevice> devices;
         for (const auto& descriptor : ListGpuDevices()) {
-            const auto& deviceName = descriptor.DeviceName;
-            if (std::find(Options_.GpuDevices.begin(), Options_.GpuDevices.end(), deviceName) == Options_.GpuDevices.end()) {
+            const auto& deviceIndex = descriptor.DeviceIndex;
```

Comment:
Just int or auto, it's a primitive type

Reply:
Created a new PR: https://github.com/ytsaurus/ytsaurus/pull/290